### ISSUE TITLE
Cross purposes

### ISSUE THREAD

**Issue by embray** [sagemath/docker#4]
We now have https://github.com/sagemath/docker and https://github.com/sagemath/docker-images, the former obviously being the predecessor.

Do we have a plan to shut down this repository in favor of the new one?  Also the question remains what to do with the existing sagemath/sage image on dockerhub.  Do we make an an alias of sagemath/sagemath?  Or do we do away with sagemath/sagemath and just override sagemath/sage which the newer image?

**Comment by nthiery** [comment]
Yes, once we will be confident enough in the new images to superseed the previous one, we probably can shut down this repository.
As for the images: I am not sure that sagemath/sagemath is backward compatible with sagemath/sage. If this is not the case, we can just leave sagemath/sage as is, with a comment in the description specifying that the image is not maintained, and pointing to the new one.

**Comment by embray** [comment]
I think it might still be confusing and would be better if we can delete it from docker hub *if possible*.  I'm not sure whether or not it's possible (I don't have permission).  But you're right that if we at least put `[DEPRECATED]` in the description that might be fine.  By the way:

```
$ docker search sage
NAME                           DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
celery                         Celery is an open source asynchronous task...   79        [OK]       
sagemath/sage                  Sage Docker Image                               12                   
mvictoras/sage2                                                                2                    [OK]
sagemath/sagemath                                                              1                    
jeromebaum/sage                                                                1                    
sagent/graylog                 Graylog server and web-interface docker image   1                    [OK]
sagent/opscenter               Docker container for running Datastax Opsc...   1                    [OK]
```

Right now `sagemath/sage` comes up before `sagemath/sagemath` which makes sense--it has been around longer and has more "stars".

Incidentally, I noticed that there's also an "OFFICIAL" column.  I don't know how we get that designation but we'll want that too.  I don't know why celery tops the list, in this case, other than that it has more stars, and is also a garnish.

**Comment by nthiery** [comment]
Here is the documentation about official repositories and how to become one:
    https://docs.docker.com/docker-hub/official_repos/
We may indeed seek to become official once our docker images will be heavily tested. It's not obvious though that our images are of general interest for that. Certainly something we can discuss informally with the contact they provide when times come.

**Comment by nthiery** [comment]
In the settings tab of sagemath/sage, there is a delete button. So, if so we wanted, it seems we indeed could delete it.

**Comment by masaeedu** [comment]
@nthiery Could you please add a note in the README pointing to the new repo, and indicating that future development is happening there? I was trying to get this repo to work with the latest docker but there's a few hiccups getting started because of outdated info in the README:

 - `--rm` and `-d` are no longer compatible flags
 - Port forwarding doesn't work on platforms with a remote Docker host, as mentioned in #2

I was just about to give up and bookmark this for later when I found the other repo (and docker image), which works well for me.
